packages/xo-server/src/api/vm.mjs

```diff
@@ -507,6 +508,9 @@ migrate.params = {
 
   // Identifier of the Network use for the migration
   migrationNetwork: { type: 'string', optional: true },
+
+  // Bypass the verification asserting whether a VM can be migrated to the specified destination.
+  bypassAssert: { type: 'boolean', optional: true },
```

Comment:
I know it's not done everywhere but it's important to do it that way for API introspection :)

```suggestion
  bypassAssert: {
    description: 'Bypass the verification asserting whether a VM can be migrated to the specified destination,
    optional: true,
    type: 'boolean',
  },
```